www/source/tutorials/build-your-own/connect-database.html.slim

```diff
@@ -0,0 +1,284 @@
+---
+title: Habitat - Connect to Database
+---
+
+section
+ h1 Connect to a Database
+
+ p As part of the parsing logic of your Gemfile, the Ruby scaffolding detects any database gems that you have added to your Gemfile and will modify the default init hook and other web application setup scripts that it creates to provide a default, but customizable configuration behavior. The connection logic provided in the scaffolding also allows you to connect your web application to a database in a number of ways such as through containers, virtual machines, database cloud services, and so on. 
+ 
+ p The Ruby scaffolding will set up this connection to logic for PostgreSQL databases assuming your Gemfile contains the following PostgreSQL database gems:
+
+ ul
+  li pg
+  li activerecord-jdbcpostgresql-adapter
+  li jdbc-postgres
+  li jdbc-postgresql
+  li jruby-pg
+  li rjack-jdbc-postgres
+  li tgbyte-activerecord-jdbcpostgresql-adapter
+
+ h2 Setting service bindings
+
+ p Service bindings allows runtime configuration values for one service group to be shared with another service group, forming a producer/consumer relationship. When you want to share out configuration data with another service group, you must add the <code>pkg_exports</code> setting to your plan.sh. The following shows how the Ruby scaffolding exports a port key for consuming services, such as a reverse proxy or load balancer.
```

Comment:
This feels like a duplication of something that should already exist largely in this format. Can we scrub the non-unique information and link there instead or use a partial?